spec/apis/hackney_api/repairs_client_spec.rb

```diff
@@ -587,7 +587,8 @@
               body: {
                 objectKey: "uhorder",
                 objectReference: "00000001",
-                text: "It's all fine"}.to_json)
+                text: "It's all fine",
+                lbhemail: "Celia" }.to_json)
```

Comment:
shouldn't there be a test for the model as well?

Reply:
There wasn't one before so I assumed it was just fine. I can make one?

Reply:
Yess, please! We should definitely test the model layer, that is the most important bit.

Reply:
Btw, when you do that, inline as much code as possible on the tests.